src/screens/Connectors/ConnectorMetaData/ApplePay/ApplePayIntegrationHelper.res

```diff
@@ -54,6 +54,83 @@ module HostURL = {
   }
 }
 
+module SampleEmail = {
+  @react.component
+  let make = () => {
+    open GlobalVars
+    let showToast = ToastState.useShowToast()
+
+    let (isTextVisible, setIsTextVisible) = React.useState(_ => false)
+
+    let businessDescription = "<One sentence about your business>. The business operates across <XX> countries and has customers across the world."
+    let featureReqText = "We are using Hyperswitch, a Level 1 PCI DSS 3.2.1 compliant Payments Orchestrator, to manage payments on our website. In addition to Stripe, since we are using other processors as well to process payments across multiple geographies, we wanted to use Hyperswitch's Payment Processing certificate to decrypt Apple pay tokens and send the decrypted Apple pay tokens to Stripe. So, please enable processing decrypted Apple pay token feature on our Stripe account. We've attached Hyperswitch's PCI DSS AoC for reference."
+
+    let emailContent = `Stripe Account id: <Enter your account id>
+
+    A detailed business description: 
+    ${businessDescription}
+
+    Feature Request:
+    ${featureReqText}`
+
+    let truncatedText = isTextVisible
+      ? featureReqText
+      : featureReqText->String.slice(~start=0, ~end=50)
+
+    let truncatedTextElement =
+      <p className="flex gap-2">
+        {truncatedText->React.string}
+        <p
+          className="cursor-pointer text-blue-400 text-xl"
+          onClick={_ => setIsTextVisible(_ => true)}>
+          {"..."->React.string}
+        </p>
+      </p>
+
+    <div className="flex flex-col">
+      <span className="mt-2 text-base  font-normal">
+        <span className="text-hyperswitch_black opacity-50">
+          {"Since the Apple Pay Web Domain flow involves decryption at Hyperswitch, you would need to write to Stripe support (support@stripe.com) to get this feature enabled for your Stripe account. You can use the following text in the email, attach our"->React.string}
+        </span>
+        <a
+          rel="noopener noreferrer"
+          className="text-blue-600 underline underline-offset-2 px-2 !opacity-100"
+          href={`${appendDashboardPath(~url="/compliance")}`}>
+          {"PCI DSS AoC certificate"->React.string}
+        </a>
+        <span className="text-hyperswitch_black opacity-50">
+          {"and copy our Support team (biz@hyperswitch.io):"->React.string}
+        </span>
+      </span>
+      <div className="border border-gray-400 rounded-md flex flex-row gap-8 p-4 mt-4 bg-gray-200">
+        <div className="flex flex-col gap-4 ">
+          <span className="flex gap-1">
+            {"Stripe Account id: <Enter your account id:you can find it"->React.string}
+            <p className="underline text-blue-400 underline-offset-1"> {"here"->React.string} </p>
+            <p> {">"->React.string} </p>
+          </span>
```

Comment:
Can you change this - 
Because p is a block level element and using flex  here is breaking the ui, it can be  simplified using span
```
        <span>
            {"Stripe Account id: <Enter your account id:you can find it "->React.string}
            <span className="underline text-blue-400 underline-offset-1">
              {"here"->React.string}
            </span>
            <span> {">"->React.string} </span>
          </span>
```
          
Before UI
   
![image](https://github.com/user-attachments/assets/fc59aa69-c055-47a9-b529-14b9c17d40c5)

After UI 

![image](https://github.com/user-attachments/assets/08f31fdd-9a76-48be-afbc-e7513ec2693c)

Reply:
Changed to <a/> as it is a reference and also <a> is also a inline element
```
 <span>
            {"Stripe Account id: <Enter your account id:you can find it "->React.string}
            <a
              className="underline text-blue-400 underline-offset-1"
              href="">
              {"here"->React.string}
            </a>
            <span> {">"->React.string} </span>
 </span>           
```

Reply:
yeah, great